NewsScraper/DawnCrawler/DawnSpiders/spiders/dawn.py

```diff
@@ -0,0 +1,61 @@
+from DawnSpiders.items import ArticleItem
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+
+class DawnSpider(CrawlSpider):
+    name = 'dawn'
+    allowed_domains = ['dawn.com']
+    start_urls = [
+        "https://www.dawn.com/"
+
+    ]
+    rules = (
+        Rule(
+            LinkExtractor(restrict_xpaths=[
+                "//*[@class='nav__item--pakistan']",
+                "//*[@class='nav__item--business']",
+                "//*[@class='nav__item--opinion']",
+                "//*[@class='nav__item--prism ']",
+                "//*[@class='nav__item--entertainment']",
+                "//*[@class='nav__item--sport']",
+                "//*[@class='nav__item--magazines']",
+                "//*[@class='nav__item--world']",
+                "//*[@class='nav__item--tech']",
+                "//*[@class='nav__item--popular ']",
+                "//*[@class='nav__item--multimedia']",
+                "//*[@class='nav__item--newspaper']",
+                "//*[@class='nav__item--in-depth ']",
+            ],
+
+            ),
+            callback='parse',
+            follow=True
+        ),
+        Rule(
+            LinkExtractor(allow='.*/news/.*'),
```

Comment:
Please dont use `allow`. Does `.story` works here?

Reply:
Sir .story is causing issues, I'm getting the author profile pages and then trying to parse them

Reply:
I'm using it with restrict_css btw